src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -271,7 +271,27 @@ export default {
 
             if (this.relatedTelemetry.hasRelatedTelemetry) {
                 isFresh = true;
-                for (let key of this.spacecraftKeys) {
+                for (let key of this.spacecraftPositionKeys) {
+                    if (this.relatedTelemetry[key] && latest[key] && focused[key]) {
+                        if (!this.relatedTelemetry[key].comparisonFunction(latest[key], focused[key])) {
+                            isFresh = false;
+                        }
```

Comment:
Can we do isFresh = Boolean(this.relatedTelemetry[key].comparisonFunction(latest[key], focused[key]))

Reply:
Fo Sho!